packages/venia-concept/src/reducers/__tests__/purchaseDetails.spec.js

```diff
@@ -0,0 +1,8 @@
+import purchaseDetails from '../purchaseDetails';
```

Comment:
please, rename import to purchaseDetailsReducer. It will be easier to read and understand the code of this file.